handbook/technical-overview/api-design-guide/documentation.md

```diff
@@ -1,14 +1,24 @@
 # Documentation
 
-API documentation should be targeted towards the developer that will consume the API. A good documentation is one of the single most important qualities of an API. It makes it much easier for other developers to use services and significantly reduces implementation time for API consumers. The API developer is responsible for keeping the documentation up to date.
+API documentation should be targeted towards the developer that will consume the API.
+A good documentation is one of the single most important qualities of an API.
+It makes it much easier for other developers to use services and significantly
+reduces implementation time for API consumers. The API developer is responsible for
+keeping the documentation up to date.
 
-To help with keeping documentation up to date consider using automatic generation tools that during build time can, for example, gather comments in predefined syntax and generate the [Open API Specification](https://swagger.io/specification/) (OAS), this means that the OAS lives bundled with the code and should be easier for developers to maintain.
+To help with keeping documentation up to date consider using automatic generation
+tools that during build time can, for example, gather comments in predefined syntax
+and generate the [Open API Specification](https://swagger.io/specification/) (OAS),
+this means that the OAS lives bundled with the code and should be easier for developers to maintain.
 
-**Note** - To be able to register a **REST** service to the _API Catalogue_ the service **MUST** provide an **OPENAPI 3** service description.
+A good Open API Specification document describes the following fields:
+
+- [`servers`](https://spec.openapis.org/oas/v3.1.0#server-object): Describing the endpoints for the API service
```

Comment:
I've heard some complaints about this property. How it should be set up when using X-Road?

The service provider can't know the URLs which a client will use to call the service since they always communicate through X-Road security servers. So this gets filled with internal URLs which may confuse client devs...

Also, should change phrasing of "A good Open API Specification" to be more direct, using e.g. can, should or must.

Reply:
Good argument. I will remove the `servers` field.